src/pages/recommend/Keyword.tsx

```
@@ -1,7 +1,37 @@
 import React from 'react';
+import { useNavigate } from 'react-router-dom';
+import { useState } from 'react';
+import type { FilterList } from '../../types/filter';
+import { initialFilterData } from '../../data/initialFilterData';
+import { Navbar, NavbarContainer } from '../../components/common/navbar';
```

ESLint

[error] 'Navbar' is defined but never used.

(@typescript-eslint/no-unused-vars)